src/Kunstmaan/TranslatorBundle/Command/ImportTranslationsCommand.php

```diff
@@ -8,8 +8,17 @@
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 
+/**
+ * @final since 6.0
```

Comment:
I think that the `@final` annotation should be on the new class and that we need to set a deprecation on this class instead. Better yet, just make the new command a final class.